core/js/setupchecks.js

```diff
@@ -223,6 +223,14 @@
 							type: OC.SetupChecks.MESSAGE_TYPE_WARNING
 						});
 					}
+					if(data.hasDBFileLocking) {
+						messages.push({
+							msg: t('core', 'The database is used for transactional file locking. This will lead to performance issues. Use redis for transactional file locking to improve the performance. See the {linkstart}documentation ↗{linkend} for more information.')
```

Comment:
Only show this when redis/memcache is there?

Reply:
You mean if redis/memcached is not used?

Reply:
If they are used in general, but not for locking.

If no redis/memcached is there, there is already a message about it, so no need to add a second one:
![image](https://user-images.githubusercontent.com/213943/232483928-3f4cf96e-4902-42c9-8481-934a624ceaf4.png)

Reply:
Ah I see. Let me investigate then

Reply:
Then let's use a less aggressive wording, because there is no performance issue with many small user-count instances

```suggestion
							msg: t('core', 'The database is used for transactional file locking. To enhance performance, please configure memcache, if available. See the {linkstart}documentation ↗{linkend} for more information.')
```

Reply:
done